eslint-config-feedzai/rules/es6.js

```diff
@@ -63,6 +63,18 @@ module.exports = {
         "template-curly-spacing": "error",
 
         // /!\ Disallow template literal placeholder syntax in regular strings
-        "no-template-curly-in-string": "warn"
+        "no-template-curly-in-string": "warn",
+
+        // require method and property shorthand syntax for object literals
+        // https://eslint.org/docs/rules/object-shorthand
```

Comment:
I think requiring this is a bit too much. I think it is still valid to use the old syntax

Reply:
Drop this one.